tests/NHSD.BuyingCatalogue.API.IntegrationTests/Steps/Entities/SolutionEntitySteps.cs

```diff
@@ -81,11 +83,18 @@ public static async Task ThenSolutionsAreLinkedToCapabilities(Table table)
             }
         }
 
+        //[Given(@"a Solution (.*) does exist")]
+        //public static async Task GivenASolutionSlnDoesExist(string solutionId)
+        //{
+        //    var solutionList = await SolutionEntity.FetchAllAsync();
+        //    solutionList.Select(s => s.SolutionId).Should().Contain(solutionId);
+        //}
+
```

Comment:
Was this added for testing? If so, it can be deleted.

Reply:
Yes it was for testing and now removed.